hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/web/ozShell/keys/GetAclKeyHandler.java

```diff
@@ -78,8 +78,8 @@ public Void call() throws Exception {
 
     List<OzoneAcl> result = client.getObjectStore().getAcl(obj);
 
-    System.out.printf("%s%n", JsonUtils.toJsonStringWithDefaultPrettyPrinter(
-        JsonUtils.toJsonString(result)));
+    System.out.printf("%s%n",
+        JsonUtils.toJsonStringWithDefaultPrettyPrinter(result));
```

Comment:
Does this need to be fixed?

Reply:
I think this should be fine, as we are not converting to String and printing.

Reply:
Isn't that what `toJsonStringWithDefaultPrettyPrinter` does - i.e. convert to string. Then we print it out.

Reply:
Yes, _toJsonStringWithDefaultPrettyPrinter_ converts to string but in Json format (with proper indentations and new line for each element). 
Previously, we were converting the object to be printed to string (using _JsonUtils.toJsonString(result)_), then converting it back into object (inside old _toJsonStringWithDefaultPrettyPrinter()_) and then again converting it into string.
```
//Old code
public static String toJsonStringWithDefaultPrettyPrinter(String jsonString)
      throws IOException {
    Object json = READER.readValue(jsonString);
    return WRITTER.writeValueAsString(json);
  }
```
The Json Injection issue happens when converting the jsonString back to Object without validation.

Reply:
Thanks for the clarification.